nvim/README.md

```
@@ -172,14 +173,36 @@ nvim/
 
 - `js/` - JavaScript
-- `ts/` - TypeScript
-- `lua/` - Lua
+- `ts/` - TypeScript (api, commander, controller, gunshi, service)
+- `lua/` - Lua module
 - `md/` - Markdown
 - `project/` - プロジェクト雛形
 
+使用例：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)